bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/niri/Niri.java

```diff
@@ -208,86 +173,101 @@ public String getDirectory() {
         return ITCConstants.LIB + "/" + INSTR_DIR;
     }
 
-    // TODO: This mimics the result of the original convoluted code. (And is probably wrong?)
-    // TODO: Verify with science and use unified getObservingStart() method from the base class.
+    // TODO: This mimics the result of the original convoluted code.
```

Comment:
I spare you the details. I was able to simplify that method, however it still overrides some behavior from the base class that is slightly different.. so no unfortunately, I still need to check if I can replace this instrument specific piece of code with the behavior from the base class... agh. Blech.